docs/concepts/validators.md

```diff
@@ -116,28 +120,67 @@ rail_str = """
 <rail version="0.1">
 
 <output>
-    <string name="a-string" type="string" format="starts-with-a">
-    <string name="custom-string" type="string" format="starts-with: my-prefix">
+    <string name="a-string" type="string" format="starts-with-a" />
+    <string name="custom-string" type="string" format="starts-with: my " />
 </output>
 
 
 <prompt>
 Generate a dataset of fake word pairs within a JSON object.
-The "a-string" property should start with the letter "a",
-and the "custom-string" property should start with "my-prefix".
 
-${guardrails.complete_json_suffix}
+${gr.complete_json_suffix_v2}
 </prompt>
 
 </rail>
 """
 
+# Initialise the Guard
 guard = Guard.from_rail_string(rail_string=rail_str)
 
+# Make the call
 raw_output, guarded_output, *rest = guard(
-    llm_api=openai.ChatCompletion.create,
-    model="gpt-3.5-turbo"
+    llm_api=openai.chat.completions.create,
+    model="gpt-3.5-turbo",
+)
+
+# See the outputs
+print(guard.history.last.tree)
+``` 
+
+**OR**
+
+### Using Pydantic
```

Comment:
Let's add two sections.  One for Pydantic 1.x (what's already here) then another for Pydantic 2.x

Reply:
Actually tested the same thing for v2, it works